Lib/test/test_new_pyc.py

```diff
@@ -142,9 +142,9 @@ def test_speed_many_globals(self):
         GLOBALS_PER_FUNC = 100
         source = []
         for f_index in range(NUM_FUNCS):
-            source.append(f"def f{f_index}():")
             for g_index in range(GLOBALS_PER_FUNC):
-                source.append(f"    global a_{f_index}_{g_index}")
+                source.append(f"a_{f_index}_{g_index} = 1")
```

Comment:
Without a ‘global’ it’s just a local, isn’t it?

Reply:
It's outside the function definition.

Before we had EXTENDED_ARGS we couldn't have NUM_FUNCS*GLOBALS_PER_FUNC such definitions in module scope, so I used "global" inside the function to get around that.